app/src/main/java/com/example/campusguide/map/GoogleMapInitializer.kt

```diff
@@ -24,13 +25,17 @@ class GoogleMapInitializer constructor(private val activity: FragmentActivity, p
             map.uiSettings.isMyLocationButtonEnabled = false
 
             // Center the map on Hall building
-            val hall = LatLng(45.497290, -73.578824)
-            map.animateCamera(CameraUpdateFactory.newLatLngZoom(hall,
-                Constants.ZOOM_STREET_LVL
-            ))
+            if(mapId == "maps_activity_map") {
```

Comment:
Why do we need that if we only have one map?

Reply:
We now have two activities with two different XML layouts and which require two different map fragments